src/app/asset-page/artstor-viewer/artstor-viewer.component.scss

```diff
@@ -146,6 +151,11 @@ $desktop: 992px;*/
     background: rgba(66,66,66,1.0);
 }
 
+.asset-viewer__edge-arrows .btn:focus {
+    outline: auto 2px Highlight !important;
```

Comment:
It's a little curious that we are manually trying to reproduce the blue boarding chrome provides. Is there a reason why we don't get this for free?

Reply:
Do other browsers implement the same border? I thought Firefox was different, could be wrong.

Reply:
Browsers do have different focus outlines but I think it would be best if we could try to keep things to be the browser default? But Evan and I were talking and noticed that open seadragon might override the default browser CSS and trying to re-override it with the default might be tricky

Reply:
Firefox's default border is dotted lines. The dotted line cannot be viewed in fullscreen mode, that's why I manually add blue border to it.

Reply:
For chrome, elements created by openseadragon and elements in the `artstor.viewer.component.html` don't get the blue border for free.

Reply:
hmm interesting. I am wondering if the default bordering is even what we are after from a UX perspective. I know on Chrome we get the blue bordering, which we are trying to replicate, but on firefox we get the dotted outline, which might not be great for accessibility. Maybe we need to talk with Nick to see if there is anything better we can do to create uniformity instead of trying to rehardcoded defaults.

Potentially unrelated, but I found some other places where we are explicitly turning off some outlines. 
https://github.com/ithaka/aiw-ui/blob/461b7b24edd955479bd240438f39de062a735ec5/src/sass/modules/_viewer.scss#L2-L6

Reply:
I noticed that for this particular css rule we are actually dealing iwth the pagination button though. I noticed that the css that is removing that border is probably related to this: https://github.com/ithaka/aiw-ui/blob/461b7b24edd955479bd240438f39de062a735ec5/src/assets/styleguide/styles/all.css#L1639

At least that is what I can see in the DOM

Reply:
@eslawski That's for non-focus state. But even we remove that line, the default border is still missing. I tried to use default borders before I turn to the solution of adding hardcoded borders, but it didn't work. I can dig more though.